psutil/_psbsd.py

```diff
@@ -432,6 +432,22 @@ def sensors_battery():
             secsleft = minsleft * 60
         return _common.sbattery(percent, secsleft, power_plugged)
 
+    def sensors_temperatures():
+        """Return systemp temperatures"""
+        ret = dict()
+        ret["coretemp"] = list()
```

Comment:
This is not correct because if there are no temperatures you will return a `{"coretemp": []}` dict which will evaluate to `True`. Instead define ret as:

`ret = collections.defaultdict(list)`

...and when you add elements to it do:

`ret["coretemp"].append(...)`

Finally, at the end of the function transform it to a plain dict with:

`return dict(ret)`